src/components/encounter/ohri-encounter-form.component.tsx

```diff
@@ -527,6 +532,10 @@ export const OHRIEncounterForm: React.FC<OHRIEncounterFormProps> = ({
     }
 
     if (encounterForSubmission.obs?.length || encounterForSubmission.orders?.length) {
+      //Save Identifiers if any
+      if (Object.keys(encounterContext?.patientIdentifier).length) {
+        saveIdentifier(patient, encounterContext.patientIdentifier);
```

Comment:
Two questions:
1. Why do we attempt to save the ID in two places?
2. What happens if the saving fails?

Reply:
Just forgot to delete this code. Cleaning up now